packages/admin-ui/ui/src/providers/feature-flag-provider.tsx

```diff
@@ -20,7 +20,7 @@ const defaultFeatureFlagContext: {
   },
 }
 
-const FeatureFlagContext = React.createContext(defaultFeatureFlagContext)
+export const FeatureFlagContext = React.createContext(defaultFeatureFlagContext)
```

Comment:
minor-nit: Use the hook instead of re-exporting this. We always want to use the hooks we create for Contexts so we can throw errors if they are used outside of the proper provider.